### PR TITLE
User auth-5-ua landing page-01

### PR DESCRIPTION
**Pull request by Robel-003**
Draft PR.. Still have to fix the css to have it replicate the demo Brandon sent. Forgot password not yet working. Sign-in and Sign-up page not yet registering user's info because db hasn't been created yet. Lots of stuff still incomplete. 
Regardless of all this, any and all feedback is greatly appreciated, thanks.

### CODE REVIEW

**Review by taylorpapke — APPROVED** [review]
I see some good results here! I went ahead and ran this on VSCode and only had to do a simple npm install and npm run dev to get the app running in a React tab. I am able to navigate to sign up page and back to sign in page via home button. I can submit a sign in given the correct input into the text fields. The only issue I see is that the text field text is the same color as the text field (white) and is not visible unless you highlight it. Also, the forgot password link is unclickable and shows a text cursor when you hover over it. In my opinion, these are minor fixes that can be easily improved upon in later iterations.

**Review by brandondombrowsky — DISMISSED** [review]
LGTM. I agree with Taylor. This is good progress!

**Comment by brinkbrink** [comment]
Thank you for the update, Robel!

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,8 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
+package.json
```
Comment (bcko) [inline .gitignore:8]: I believe we want to check both package-lock.json and package.json to our repository. This way we all share the same package related json files.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "eslint": "8.43.0",
     "eslint-config-next": "13.4.7",
     "next": "13.4.7",
+    "next-auth": "^4.22.1",
```
Comment (bcko) [inline package.json:19]: Let's pin next-auth to a specific version

File: tsconfig.json [hunk 3]
```diff
@@ -23,6 +23,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.ts", ".next/types/**/*.ts"],
```
Comment (bcko) [inline tsconfig.json:26]: `**/*.ts` already exists so let's undo this change.

**Comment by bcko** [comment]
Feel free to convert this from a draft pull request to a regular pull request.
Also don't forget to rebase your branch to the most recent `main` branch so you can resolve conflicts.

**Review by taylorpapke — APPROVED** [review]
Went ahead and followed BC's steps to rebase branch with main. Deleted conflicted package-lock.json and node_modules before reinstalling. Should be ok now as far as conflicts but there are a few convos that still need to be resolved.

File: .gitignore (hunk 1 above)
Comment (Robel-003) [reply .gitignore]: fixed..

File: tsconfig.json (hunk 3 above)
Comment (Robel-003) [reply tsconfig.json:26]: fixed

File: package.json (hunk 2 above)
Comment (Robel-003) [reply package.json]: fixed

**Review by brinkbrink — APPROVED** [review]
LGTM

**Comment by brinkbrink** [comment]
Just need another review to get this merged!

**Comment by tinpham5614** [comment]
From my perspective, everything seems to be functioning properly. I have tested the home navigation, forgot password, and sign-up functions working correctly, redirecting users to the appropriate pages. The input text color matches the input field color, which may be difficult to see but can be adjusted later on.

**Review by tinpham5614 — APPROVED** [review]

**Review by taylorpapke — COMMENTED** [review]
Went ahead and tested it one last time and it is still working the same.
